packages/rpm/kubelet.spec

```diff
@@ -39,9 +39,6 @@ Requires: ethtool
 Requires: iproute
 Requires: ebtables
 Requires: conntrack
-Obsoletes: kubernetes-cni
-Conflicts: kubernetes-cni
```

Comment:
The same problem stays as for Debian. How upgraded should be handled in this case?

Reply:
The kubernetes-cni package stays installed I guess. I'm wondering if this is a problem, users might have to cleanup those stale package afterwards.

Reply:
I think you want to keep the `Conflicts` but remove `Obsoletes`.
Conflicts tells yum that both kubelet v1.18.4 and kubernetes-cni cannot be installed.
Obsoletes prevents kubernetes-cni to be installed at all even with older versions of the kubelet, because it says that anytime someone is trying to install kubernetes-cni, they should install kubelet v1.18.4.

Reply:
Thanks for the hint, fixed as suggested.